schemas/aerleon-definition-yaml.schema.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$comment": "https://aerleon.readthedocs.io/en/latest/reference/naming/",
+  "title": "Aerleon Definition File",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "networks": {
+      "description": "An Aerleon definition file defines named networks and services which can be referenced by Aerleon Policy files. Networks are lists of IP addresses or CIDR IP address ranges. Services are lists of port/protocol pairs (e.g. tcp/80).\nDefinition files must include a \"networks\" section, a \"services\" section, or both.\nNetworks and services may use composition in their definitions. Include another definition by creating an item containing the \"name\" property.",
+      "type": "object",
+      "additionalProperties": { "$ref": "#/$defs/networkDefinition" },
+      "propertyNames": { "$ref": "#/$defs/token" }
+    },
+    "services": {
+      "description": "An Aerleon definition file defines named networks and services which can be referenced by Aerleon Policy files. Networks are lists of IP addresses or CIDR IP address ranges. Services are lists of port/protocol pairs (e.g. tcp/80).\nDefinition files must include a \"networks\" section, a \"services\" section, or both.\nNetworks and services may use composition in their definitions. Include another definition by creating an item containing the \"name\" property.",
+      "type": "object",
+      "additionalProperties": { "$ref": "#/$defs/serviceDefinition" },
+      "propertyNames": { "$ref": "#/$defs/token" }
+    }
+  },
+  "$defs": {
+    "token": {
+      "type": "string",
+      "pattern": "^[-_A-Z0-9]+$"
+    },
+    "comment": {
+      "description": "When inserting a network or service value you can attach a comment directly to that value. The comment will be loaded and potentially included in generated output on platforms that support it.",
+      "type": "string"
+    },
+    "address": {
+      "description": "When defining a network you can insert data as a specific IP address or a CIDR IP address range expression.",
```

Comment:
Can't this also be a reference to another network definition?

Reply:
That would need to go in the `name:` field, and this is a description just for the `address:` field. The description one level above, on `values:`, does explain that you can have both in your network definition.